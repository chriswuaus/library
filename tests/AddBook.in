ADD BOOK sansd.csv 111111
ADd book test.csv 69696996
add book test.csv 111111
add book test.csv 111111
exit
