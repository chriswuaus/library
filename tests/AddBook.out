user: No such file.

user: No such book in file.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Book already exists in system.

user: Ending Library process.
